.github/workflows/tests-robotframework-intellij.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       with:
         java-version: 11
     - name: Install deps
-      run: pip install --upgrade pip fire twine wheel setuptools robotframework
+      run: pip install --upgrade pip fire twine wheel setuptools robotframework robotremoteserver
```

Comment:
I don't think you're testing the `robotremoteserver` with the `Intellij` plugin (which is ok), but then, this shouldn't be needed (i.e.: this file is for the tests related to the `Robot Framework Language Server Intellij Plugin` ..